README.md

```diff
@@ -19,6 +19,18 @@ Cargo.toml:
 onvif-rs = "0.1"
 ```
 
+## Build
```

Comment:
I think this is more a troubleshooting section than build. Wouldn't you agree?

Reply:
Makes sense